packages/connect-migrate/src/transforms/modify-imports.spec.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { join } from "node:path";
+import { readFileSync, readdirSync } from "node:fs";
+import jscodeshift from "jscodeshift/src/core";
+import transform from "./modify-imports";
+
+// Make sure to import the testfixtures from source so we're not comparing transformed fixtures
+const fixtureLocation = join(
+  __dirname,
+  "../../../src/transforms",
+  "__testfixtures__",
+);
```

Comment:
I looked into ways to run codeshift the same way in tests and in the CLI, but the workers do give us a significant bump in performance for large numbers of files.

Since we can't run the same code paths in tests as we do in the CLI, we can just as well remove the `__testfixtures__` directory and work with strings here in the tests. 

The we end up with something like this:

```ts
import jscodeshift from "jscodeshift";
import transform from "./modify-imports.js";

function t(source: string) {
  const shift = jscodeshift.withParser("tsx");
  return transform(
    { path: "test-file", source },
    {
      jscodeshift: shift,
      j: shift,
      stats: () => {},
      report: () => {},
    },
    {},
  );
}

describe("modify-imports", () => {
  it("should modify import", () => {
    const got = `import a from "@bufbuild/connect";`;
    const want = `import a from "@connectrpc/connect";`;
    expect(t(got)?.trim()).toBe(want.trim());
  });
  it("should modify import with single quotes", () => {
    const got = `import a from '@bufbuild/connect';`;
    const want = `import a from '@connectrpc/connect';`;
    expect(t(got)?.trim()).toBe(want.trim());
  });
  it("should modify subpath import", () => {
    const got = `import a from "@bufbuild/connect/protocol";`;
    const want = `import a from "@connectrpc/connect/protocol";`;
    expect(t(got)?.trim()).toBe(want.trim());
  });
  it("should not modify irrelevant import", () => {
    const got = `import a from "@foobar/connect";`;
    const want = `import a from "@foobar/connect";`;
    expect(t(got)?.trim()).toBe(want.trim());
  });
  it("should modify require", () => {
    const got = `const a = require("@bufbuild/connect");`;
    const want = `const a = require("@connectrpc/connect");`;
    expect(t(got)?.trim()).toBe(want.trim());
  });
  it("should not modify irrelevant require", () => {
    const got = `const a = require("@foobar/connect");`;
    const want = `const a = require("@foobar/connect");`;
    expect(t(got)?.trim()).toBe(want.trim());
  });
  it("should modify require with single quotes", () => {
    const got = `const a = require('@bufbuild/connect');`;
    const want = `const a = require('@connectrpc/connect');`;
    expect(t(got)?.trim()).toBe(want.trim());
  });
});
```

I believe we don't lose any confidence with these tests (it's clear now that the file name is irrelevant), but we can remove a lot of files and ignores.

What do you think?

Reply:
Yeah that sounds fine to me.

Reply:
Adding it in https://github.com/connectrpc/connect-es/pull/774